libs/hwes/event.go

```diff
@@ -104,6 +146,21 @@ func NewEvent(aggregate Aggregate, eventType string, opts ...EventOption) (Event
 	return evt, nil
 }
 
+func NewEventFromProto(aggregate Aggregate, message proto.Message, opts ...EventOption) (Event, error) {
```

Comment:
```suggestion
func FromProto(aggregate Aggregate, message proto.Message, opts ...EventOption) (Event, error) {
```

Reply:
We create a new event from the passed proto message. I like the verbosity there. It also indicates a "constructor" like method.